docs/guidelines/components/tables.md

```diff
@@ -0,0 +1,202 @@
+                <input type="checkbox">
+            </td>
+            <td>1</td>
+            <td>English (United Kingdom)</td>
+            <td>Administrator User</td>
+            <td>
+                <button type="button" class="btn btn-icon">
+                    <svg class="ez-icon ez-icon-edit">
+                        <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="../../ez-icons.svg#archive-restore"></use>
+                    </svg>
+                </button>
+            </td>
+        </tr>
+    </tbody>
+</table>
+code_example]]
+</div>
+
+##<div class="mgt-3 header-line">Secondary table</div>
+<div class="ez-guidelines-tables__with-header mgb-5">
+[[code_example {html}
+<div class="ez-table-header ground-base">
+    <div class="ez-table-header__headline">Location Content Swap</div>
+</div>
+<table class="table ez-table--no-border">
+    <tbody>
+        <tr>
+            <td>Swap the content item at this location with another
+                <button type="button" class="btn btn-outline-secondary ml-5">Select content item</button>
+            </td>
+        </tr>
+    </tbody>
+</table>
+code_example]]
+</div>
+
+##<div class="mgt-3 header-line">Notifications table</div>
+<div class="ez-guidelines-tables__notifications mgb-5">
+[[code_example {html}
+<table class="table n-table--notifications">
+    <thead>
+        <tr>
+            <th>Type</th>
+            <th>Description</th>
+            <th>Date</th>
+        </tr>
+    </thead>
+    <tbody class="n-table__body">
+        <tr class="n-notifications-modal__item fw-notification">
+            <td class="n-notifications-modal__type">
+                <span class="type__icon">
+                    <svg class="ez-icon ez-icon--review">
+                        <use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="../../ez-icons.svg#review"></use>
+                    </svg>
+                </span>
+                <span class="type__text">
+                    Content review request
+                </span>
+            </td>
+            <td class="n-notifications-modal__description">
+                <p class="description__title">
+                    From:
+                    <span class="description__title__item">Administrator user</span>
+                </p>
+                <p class="description__text">Can you check this? Thxs!</p>
+                <span class="description__read-more">Read more &raquo;</span>
+            </td>
+            <td class="n-notifications-modal__time">
+                September 13, 2018 14:58
+            </td>
+        </tr>
+    </tbody>
+</table>
+code_example]]
+</div>
+
+!!! Note
+    This table is displayed within `.n-notifications-modal` modal. Styling relies on this class.
+
+
+##<div class="mgt-3 header-line">List table</div>
+<div class="ez-guidelines-tables__list mgb-5">
+[[code_example {html}
+<table class="table ez-table--list">
+    <thead>
+        <tr>
+            <th colspan="2">
```

Comment:
In general, the structure of `ez-table--list` will be updated very soon.

Reply:
Please refer to this PR: https://github.com/ezsystems/ezplatform-admin-ui/pull/614

Reply:
So based on my previous comment, I'd add this new way in the upcoming 2.3 dev-doc branch, but again I need confirmation.